UKHomeOffice/cop-ui
# COP-3825 Remove outdated package

## AC

Resolve warnings of componentWillMount & similar

## Updated

Determined one source of these was `react-loader-advanced` package which looks to be no longer maintained (last update over 2yrs ago)

It was only used on the `DisplayForm` component to show the `ApplicationSpinner` as `submitting`

I've removed the `Loader` component and replaced it with the ternary that is used elsewhere to determine whether to show spinner or not. The ternary return statement is marked to be reviewed and potentially refactored at a later date.

## To test

- pull this branch
- remove `package-lock` & `node-modules`
- from the client folder run `npm i`
- open a form
- submit the form
- > you should see a spinner with the text 'submitting form'
